modules/storages/app/models/storages/project_storage.rb

```diff
@@ -62,11 +62,22 @@ def automatic_management_possible?
 
   def project_folder_path
```

Comment:
Gooooooood question. `fancy_folder_path_mcfancypaths` sounds a bit long. xD We use this only on the sync jobs and the dependent services.

So we could rename it with little impact overall. Let's say someone invokes this outside of the scope of an AMPF, what would be the most innocuous return?

Reply:
one option would be to return just that: the path it would(!) have if AMPF is enabled. But then a clear naming would be helpful.

Or we make it sensitive to the project folder mode, returning the real manual folder and `nil` on `inactive`. But there is no use case, so I'd hesitate to put the extra effort

Or, we check for AMPF inside there and fail loudly. That would kind of forbid using this method outside of the correct context.

I tend to option 1 or 3, a bit more leaning towards 1.

Reply:
I like 1 more than the others, but what to name the method? xD

Reply:
no big fan of my long name, are ye? :D

so, what about just `managed_project_folder_path`? skipped one word, as a dev you still know that this is a path the system is involved into.